app/views/play/CampaignView.coffee

```diff
@@ -479,14 +479,15 @@ module.exports = class CampaignView extends RootView
       return @promptForSubscription campaignSlug, 'premium campaign visited'
 
   promptForSignup: ->
-    return if features.noAuth
+    return if features.noAuth or @terrain is 'game-dev-hoc'
 
     @endHighlight()
     authModal = new CreateAccountModal supermodel: @supermodel
     authModal.mode = 'signup'
     @openModalView authModal
 
   promptForSubscription: (slug, label) ->
+    return console.log('Game dev HoC does not encourage subscribing.') if @terrain is 'game-dev-hoc'
```

Comment:
Who is this console.log for?

Reply:
Anyone who might try to be figuring out why CampaignView is acting strangely.